test-requirements.txt

```diff
@@ -6,6 +6,8 @@ colorama<0.4
 # coverage 4.4.2 broke compatibility with coverage-config-reload-plugin
 # and thus broke https://github.com/jayvdb/coverage_env_plugin .
 # See https://github.com/jayvdb/coverage_config_reload_plugin/issues/1
+git+https://github.com/coala/coala@master
+git+https://github.com/coala/coala-bears@master
```

Comment:
Undo here as well.

Reply:
We need to pull the latest coala bears repo from master so we are in dicussions to see whether it works.

Reply:
>We need to pull the latest coala bears repo from master so we are in dicussions to see whether it works.

Yes, I know that very well. To implement that we shouldn't do modifications to the test-requirements since we'll have to do a lot of changes to Dockerfile which would eventually increase the size of the image. Since we are only using it (coala-bears) on travis we should install them separately (this is what we've already done on travis, we just need to follow the same for semaphore).